docs/install.md

```diff
@@ -0,0 +1,203 @@
+
+### Enable TLS for dashboard access via Ingress
+**Will only work in the cluster node**
+#### Pre-requisites:
+1. Tekton pipelines & dashboard installed
+2. dashboard repo cloned
+
+#### Steps:
+1. Edit `ingress/ingress-https-setup.sh` with all the necessary info
+2. Run the script from within the dashboard repo
+3. Access dashboard via `https://tekton-dashboard.<IP_ADDRESS>.nip.io`
+
+
+## Install on Minishift
+
+Either follow the instructions for OpenShift above or use the operator install as per the instructions below.
+
+1. Install [tektoncd-pipeline-operator](https://github.com/openshift/tektoncd-pipeline-operator#deploy-openshift-pipelines-operator-on-minikube-for-testing)
+2. [Checkout](https://github.com/tektoncd/dashboard/blob/master/DEVELOPMENT.md#checkout-your-fork) the repository
+
+If you want to install the Dashboard into the tekton-pipelines namespace:
+
+- Install the Dashboard `./minishift-install-dashboard.sh`
+
+If you want to install the Dashboard into any other namespace:
+
+- Install the Dashboard `./minishift-install-dashboard.sh -n {NAMESPACE}`
+
+3. Wait until the pod `tekton-dashboard-1` is running in the namespace the Dashboard is installed into
+
+## Accessing the Dashboard
+
+The Dashboard can be accessed through its ClusterIP Service by running `kubectl proxy`. Assuming tekton-pipelines is the install namespace for the Dashboard, you can access the web UI at `localhost:8001/api/v1/namespaces/tekton-pipelines/services/tekton-dashboard:http/proxy/`.
+
+An alternative way to access the Dashboard is using `kubectl port-forward` e.g. if you installed the Tekton Dashboard into the `tekton-pipelines` namespace (which is the default) you can access the Dashboard with `kubectl --namespace tekton-pipelines port-forward svc/tekton-dashboard 9097:9097` and then just open `localhost:9097`.
+
+## Accessing the Dashboard on Minishift
+
+The Dashboard can be accessed by running `kubectl --namespace tekton-pipelines port-forward svc/tekton-dashboard 9097:9097`
+If installed into a namespace other than tekton-pipelines then the Dashboard can be accessed by running `kubectl --namespace $NAMESPACE port-forward svc/tekton-dashboard 9097:9097`
+You can access the web UI at `http://localhost:9097/`
+
+## Uninstalling the Dashboard on Minishift
+
+The Dashboard can be uninstalled on Minishift by running the command `./minishift-delete-dashboard.sh` Use `-n {NAMESPACE}` on the end of the command if installed into a namespace other than `tekton-pipelines`
+
+## Troubleshooting
+
+Keep in mind that When running your Tekton Pipelines, if you see a `fatal: could not read Username for *GitHub repository*: No such device or address` message in your failing Task logs, this indicates there is no `tekton.dev/git` annotated GitHub secret in use by the ServiceAccount that launched this PipelineRun. It is advised to create one through the Tekton Dashboard. The annotation will be added and the specified ServiceAccount will be patched.
```

Comment:
not sure if this belongs in install.md but don't have a better suggestion right now